website/source/api/acl/acl.html.md

```diff
@@ -263,3 +263,112 @@ agent_prefix "" {
    policy = "read"
 }
 ```
+
+## Login to Auth Method
+
+This endpoint was added in Consul 1.5.0 and is used to exchange an [auth
+method](/docs/acl/acl-auth-methods.html) bearer tokens for a newly-created
+Consul ACL token.
+
+| Method | Path                         | Produces                   |
+| ------ | ---------------------------- | -------------------------- |
+| `POST` | `/acl/login`                 | `application/json`         |
+
+The table below shows this endpoint's support for
+[blocking queries](/api/index.html#blocking-queries),
```

Comment:
```suggestion
[blocking queries](/api/features/blocking.html),
```

Reply:
I think this change should be done independently, as master still lists a whole bunch of references to the anchor:
```
consul (master)$ git grep "/api/index.html#blocking-queries" website/source | wc -l
128
```